Source/FullScreenPokemon.ts

```diff
@@ -2775,6 +2775,16 @@ module FullScreenPokemon {
                     }
                 },
                 steps * speed);
+
+            // allow the user to move the character after the jump 
+            thing.FSP.TimeHandler.addEvent(
+                function (): void {
+                    if (thing.player) {
```

Comment:
If the event won't do anything for non-players, it's best to only add it if they're a player. This way we have fewer closures being added & timed everywhere.